twilio_integration/twilio_integration/doctype/whatsapp_message/whatsapp_message.py

```diff
@@ -103,8 +116,11 @@ def store_whatsapp_message(
 		communication=None,
 		template_sid=None,
 		content_variables=None,
+		queue=False,
+		notification_type=None,
 	):
 		sender = frappe.db.get_single_value('Twilio Settings', 'whatsapp_no')
+		status = 'Queued' if queue else 'Not Sent'
```

Comment:
Not Sent means it's queued from our side
Queued is not our status, it's determined by Twilio